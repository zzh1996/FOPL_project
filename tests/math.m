Begin
    Print Plus 1 10
    Print Minus 5 3
    Print Mult 7 9
    Print Div 10 3
    Print Mod 17 6
    If Lt 1 2 Begin Print 1 End Else Begin Print 2 End
    If Gt 1 2 Begin Print 1 End Else Begin Print 2 End
    If Eq 1 2 Begin Print 1 End Else Begin Print 2 End
    If And Lt 1 2 Lt 2 1 Begin Print 1 End Else Begin Print 2 End
    If Or Lt 1 2 Lt 2 1 Begin Print 1 End Else Begin Print 2 End
    If Negb Lt 1 2 Begin Print 1 End Else Begin Print 2 End
End
