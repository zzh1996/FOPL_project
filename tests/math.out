11 2 63 3 5 1 2 2 2 1 2
